src/js/dates.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * DATES
+ *
+ * @package uri-component-library
+ */
+
+( function() {
+	'use strict';
+
+	window.addEventListener( 'DOMContentLoaded', init, false );
+
+	function init() {
+		let i;
+
+		const els = document.querySelectorAll( '.cl-date' );
+
+		for ( i = 0; i < els.length; i++ ) {
+			setupDate( els[ i ] );
+		}
+	}
+
+	function setupDate( el ) {
+		el.classList.add( 'has-js' );
+		const p = el.querySelector( '.cl-date-content-wrapper' );
+		p.addEventListener( 'click', dialogue.bind( null, el ), false );
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['dialogue' was used before it was defined.](https://app.codacy.com/manual/uriweb/uri-component-library/pullRequest?prid=4835245)